### PR TITLE
ADBP-2353

### PR DESCRIPTION
**Pull request by KashifM101**
Link to Jira:
https://101digital.atlassian.net/browse/ADBP-2353

### CODE REVIEW

File: src/components/qr-transfer-component/qr-inbound-transfer/index.tsx [hunk 1]
```diff
@@ -163,15 +163,15 @@ const QrInboundTransferComponent: React.FC<QrInboundTrasnferComponentTypes> = (
...
+          <MoneyReceiveIcon size={20} color={colors.primaryButtonLabelColor} />
           <Text
             style={[
               innerStyles.buttonText,
               {
                 fontFamily: fonts.semiBold,
-                color: colors.secondaryButtonLabelColor,
+                color: colors.primaryButtonLabelColor,
               },
             ]}
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-inbound-transfer/index.tsx:176]: why inline styles ?

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 2]
```diff
@@ -2,10 +2,10 @@ import AppLockComponent from "../../../app-lock-component";
...
 import React, {useContext, useMemo, useRef, useState} from "react";
 import {ScrollView, StyleSheet, Text, TextInput, TouchableOpacity, TouchableWithoutFeedback, View,} from "react-native";
-import {Button, ThemeContext} from "react-native-theme-component";
+import {Button, ThemeContext,useADBCurrencyFormat} from "react-native-theme-component";
 import {ArrowRightBoldIcon} from "../../../../assets/icons";
 import DangerIcon from "../../../../assets/icons/danger-icon";
-import {DecimalFormatter, maskString} from "../../../../helpers";
+import {DecimalFormatter, maskString,NumberFormatter} from "../../../../helpers";
 import Selector from "../../../selector-component";
 import RnRangeSlider from "../../components/range-slider";
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:10]: plz import all icons from icon.index file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/qr-transfer-component/qr-inbound-transfer/index.tsx (hunk 1 above)
Comment (KashifM101) [reply src/components/qr-transfer-component/qr-inbound-transfer/index.tsx:176]: i'll restructure it.

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx (hunk 2 above)
Comment (KashifM101) [reply src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:10]: ok

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**